msb/zmq_base/config.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from msb.config.MSBConfig import MSBConf
+
+
+class ZMQConf(MSBConf):
+    verbose: bool = True
```

Comment:
This parameter should not be set here. It is defined by MSBConf and should default to False

Reply:
fixed it.